javascript-challenges.test.js

```diff
@@ -39,5 +86,26 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+function toBeOrNotToBeFullHouse(hand) {
+    if (hand.length !== 5) { //if length of array is not equel to 5 elements then return false
+      return false;
+    }
+  
+    const numCounts = {}; // creating empty object 
+  
+    for (const num of hand) {
+      numCounts[num] = (numCounts[num] || 0) + 1; //iterating through each hand array, if element is already in object created, it will add 1
+    }
+  
+    const values = Object.values(numCounts); // 
+  
+    return values.includes(2) && values.includes(3); // 
+  }
 
-// Pseudo code:
+// Pseudo code: 
+// create a function named --> toBeOrNotToBeFullHouse
+//input: an array
+//output: a boolean, whether or not the array given dispays a full house
+// using conditional statment
+// using boolean
+// using chatGPT I got the following 
```

Comment:
While chatGPT is a tool developers do use in the field, it is important to understand how to solve these problems without AI, and use it to help with specific pieces.  Many companies have strict policies in using AI as you do not want proprietary information being passed to it.  That being said, using it to get unstuck can be helpful, but copying directly from it is only hurting yourself in the long run.